subsys/mpsl/fem/mpsl_fem.c

```diff
@@ -114,6 +114,30 @@ static int ppi_channel_alloc(uint8_t *ppi_channels, size_t size)
 	return 0;
 }
 
+#if defined(CONFIG_SOC_SERIES_NRF53X)
+static int egu_instance_alloc(uint8_t *p_egu_instance_no)
+{
+	// Always return EGU0.
+	*p_egu_instance_no = 0;
+
+	return 0;
+}
+
+static int egu_channel_alloc(uint8_t *egu_channels, size_t size, uint8_t egu_instance_no)
+{
+	(void)egu_instance_no;
+
+	uint8_t starting_channel = 4U;
```

Comment:
Where did the `4U` come from?

Reply:
Currently nrf 802.15.4 is the only user of egu. We use egu channels 0...3 and 15, this is the way to hardcode channels 4, 5 ...

Reply:
@ankuns could you provide some background for this number?

Reply:
I'll add a comment there.